Util/Schema/isdouble.m

```diff
@@ -0,0 +1,3 @@
+function [bool] = isdouble(arg),
```

Comment:
Do we really need all these functions (11 already!)? A bit overkill when isa() already is a one liner. I'm just a bit concerned that we'll end up with crazy amount of new .m files in the repo if we follow this practice consistently.

Reply:
It's not my fault that matlab requires functions to be in separated files if you want to use then outside of the current function scope. The main reason, however, is that I need a static function handle for type definition (see detectType). 

The only way to do that is to define a proper function, so the function handle to it is static and comparable. Moreover, these other types do not have proper signatures such as ischar.

Example:

In detectType(A) we would define:
```
isdouble = @(x) isa(x,'double')
...
Atype = @isdouble
```

In detectType(B) we would do:
```
isdouble = @(x) isa(x,'double')
...
Btype = @isdouble
```

Comparison scope would fail, because dynamical function handles are assigned to different addresses in memory:
```
Atype = detectType(A)
BType = detectType(B)
isequal(Atype,Btype) == false
```

When, I define a proper/static function, the above works in any scope:

```
 Atype = @isdouble
 Btype = @isdouble
 isequal(Atype,Btype) == true
```

Reply:
Could you declare your function isdouble etc in detectType instead of in separate .m files for each of them?

Reply:
In general: No. One may need them in any scope, not just detectType. The function handle of that function is also an output. Hence, it should easily reachable.

Specifically, they are used in testSchema. For example for input and expected output when testing. They cannot be dynamic function handles otherwise the comparison would fail.